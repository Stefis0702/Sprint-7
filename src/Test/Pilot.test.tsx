import PilotsList from "../components/Pilots/Pilots";

import{describe,it, expect} from 'vitest';

describe('PilotsList',()=>{
    it('should be a function',()=>{
        expect( PilotsList).toBeTypeOf('function');
    });
    it('should render correctly',()=>{
        expect(PilotsList).toMatchSnapshot();
    });
    
    
})