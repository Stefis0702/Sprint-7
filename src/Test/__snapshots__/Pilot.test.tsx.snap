// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HeaderCom > should render correctly 1`] = `[Function]`;

exports[`PilotsList > should render correctly 1`] = `[Function]`;
